src/core/namespace.c

```diff
@@ -1643,7 +1643,7 @@ static int apply_one_mount(
                 if (try_again)
                         r = mount_nofollow_verbose(LOG_DEBUG, what, mount_entry_path(m), NULL, MS_BIND|(rbind ? MS_REC : 0), NULL);
                 if (r < 0)
-                        return log_error_errno(r, "Failed to mount %s to %s: %m", what, mount_entry_path(m));
+                        return log_debug_errno(r, "Failed to mount %s to %s: %m", what, mount_entry_path(m));
```

Comment:
same for this one please

Reply:
why? i don't get why you want duplicate logging for all these things?

this will be logged again, with a pretty similar and more useful message, including the mount part with the prefix removed. why log this twice?

this is not how we do logging in systemd, really.